configure.ac

```diff
@@ -1,5 +1,5 @@
-AC_PREREQ([2.60])
-AC_INIT([Haskell unix package], [2.0], [libraries@haskell.org], [unix])
+AC_PREREQ([2.71])
```

Comment:
2.71 sounds a bit extreme, plenty systems use 2.69 by default. Could it be downgraded?